src/components/MostRecentScoreboard.tsx

```diff
@@ -2,25 +2,46 @@ import * as React from "react";
 import { Play } from "../types";
 import ScoreboardCard from "./ScoreboardCard";
 import styles from "./MostRecentScoreboard.module.css";
+import InfiniteScroll from "react-infinite-scroll-component";
+import LoadingCircle from "./LoadingCircle";
 
 interface MostRecentScoresProps {
   plays: Play[];
 }
 
 function MostRecentScoreboard(props: MostRecentScoresProps) {
   const { plays } = props;
-  const mostRecentScores = plays.reduce((acc, x) => {
-    // Only keep the first play for each sport
-    if (!acc[x.sport]) {
-      acc[x.sport] = x;
-    }
-    return acc;
-  }, {} as Record<string, Play>);
+  const [items, setItems] = React.useState(
+    plays.slice(0, 6).map((play, i) => <ScoreboardCard play={play} key={i} />)
+  );
+
+  const fetchData = () => {
+    setItems(
+      items.concat(
+        plays
+          .slice(items.length, items.length + 6)
+          .map((play, i) => (
+            <ScoreboardCard play={play} key={i + items.length} />
+          ))
+      )
+    );
+  };
```

Comment:
Slice and map two different ways feels redundant, but I do want to initialize to first six elements. Could be a better way.

Reply:
Eh it's fine. You could always resort to a for loop if you were _really_ worried about it, but you shouldn't be.

```js
let itemsToshow = [];
for (let i = 0; i < items.length; i++) {
  const item = <ScoreboardCard play={plays[i]} key={i} />;
  itemsToShow.push(item);
}
// untested, not sure if works
```